### PR TITLE
Run3-alca228 Update for isolated bunch selector AlCaReco

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update for isolated bunch selector AlCaReco

#### PR validation:

Useful for Reprocessing 2022 data

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

To be back ported to 12_4_X

### CODE REVIEW

**Comment by bsunanda** [comment]
@mariadalfonso Just submitted the PR and I am requesting a back port PR as well

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalNoise_cff.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import HLTrigger.HLTfilters.hltHighLevel_cfi
 noiseHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
-    HLTPaths = ['HLT_MET25'],
+    HLTPaths = ['HLT_PFMET130'],
 #    eventSetupPathsKey='HcalCalNoise',
```
Comment (tvami) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalNoise_cff.py:7]: This is not what we should do.... what we should do is the commented out line, i.e. have
`eventSetupPathsKey='HcalCalNoise'` active.

File: Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc [hunk 2]
```diff
@@ -145,7 +143,7 @@ void AlCaIsolatedBunchSelector::fillDescriptions(edm::ConfigurationDescriptions&
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("triggerResultLabel", edm::InputTag("TriggerResults", "", "HLT"));
   desc.add<std::string>("processName", "HLT");
-  desc.add<std::string>("triggerName", "HLT_HcalIsolatedBunch");
+  desc.add<std::string>("triggerName", "HLT_ZeroBias_IsolatedBunches");
```
Comment (malbouis) [inline Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc:146]: maybe this trigger bit could be dropped here and use instead the alcarecotriggerbits tag in here: https://github.com/cms-sw/cmssw/blob/master/Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsolatedBunchSelector_cff.py#L8

**Comment by tvami** [comment]
@cmsbuild , please abort

File: Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc (hunk 2 above)
Comment (bsunanda) [reply Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc:146]: I am now using only the trigger bits in the new commit

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalNoise_cff.py (hunk 1 above)
Comment (bsunanda) [reply Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalNoise_cff.py]: Done

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by tvami** [comment]
test parameters:
  - workflows = 1001.3,1002.3

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+alca
 * the ALCARECO is successfully produced: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa44c5/26956/runTheMatrix-results/1001.3_RunSingleMuon2022B+RunSingleMuon2022B+TIER0EXPRUN3+ALCAEXPRUN3+ALCAHARVDEXPRUN3/HcalCalNoise.root
 * Its content looks good as well

**Comment by tvami** [comment]
urgent
 * backport affects data taking / rereco as well

**Comment by perrotta** [comment]
+1

**Pull request merged**